integreat_cms/cms/constants/machine_translation_budget.py

```diff
@@ -0,0 +1,27 @@
+"""
+This module contains the available choices for machine translation budget
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Final
+
+
+MINIMAL: Final = 50000
+XS: Final = 100000
+S: Final = 150000
+M: Final = 250000
+L: Final = 1000000
+XL: Final = 5000000
+
+CHOICES: Final[list[tuple[int, str]]] = [
+    (MINIMAL, "50,000"),
```

Comment:
```suggestion
    (MINIMAL, "50.000"),
```
I think we should stick with the German convention in this case, because so far mostly German municipalities use the CMS. The German convention separates via dot instead of comma :)